test/extended/default_builds_test.go

```diff
@@ -0,0 +1,80 @@
+// +build default
+
+package extended
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	. "github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("STI build with .sti/environment file", func() {
+	defer GinkgoRecover()
+
+	var imageStreamFixture = filepath.Join("..", "integration", "fixtures", "test-image-stream.json")
+	var stiEnvBuildFixture = filepath.Join("fixtures", "test-env-build.json")
+	var oc = exutil.NewGinkoCLI("build-sti-env", kubeConfigPath())
```

Comment:
This test should not run as a `cluster-admin`.  You can use your `cluster-admin` rights to create a new project, assign an admin, create an editor, and then run the rest of the tests as the editor.